### PR TITLE
Feature/shoot fuel command

### PR DESCRIPTION
**Pull request by SethNguyenXBOT**

### CODE REVIEW

**Comment by WasabiFan** [comment]
What's the use of these commands? Why do this over a single command that operates on both sides?

**Review by aschokking — APPROVED** [review]

File: Competition2017/src/competition/subsystems/shoot_fuel/RightShootFuelCommandGroup.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import competition.subsystems.agitator.commands.IntakeAgitatorCommand;
+import competition.subsystems.shooter_belt.ShooterBeltsManagerSubsystem;
+import competition.subsystems.shooter_belt.commands.RunBeltIfWheelAtSpeedCommand;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem.TypicalShootingPosition;
+import competition.subsystems.shooter_wheel.commands.RunShooterWheelsForRangeCommand;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.properties.XPropertyManager;
+
+public class RightShootFuelCommandGroup extends CommandGroup{
```
Comment (aschokking) [inline Competition2017/src/competition/subsystems/shoot_fuel/RightShootFuelCommandGroup.java]: nit, add space before ` {`

File: Competition2017/src/competition/subsystems/shoot_fuel/ShootFuelCommandGroup.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import competition.subsystems.agitator.commands.IntakeAgitatorCommand;
+import competition.subsystems.shooter_belt.ShooterBeltsManagerSubsystem;
+import competition.subsystems.shooter_belt.commands.RunBeltIfWheelAtSpeedCommand;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem.TypicalShootingPosition;
+import competition.subsystems.shooter_wheel.commands.RunShooterWheelsForRangeCommand;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.properties.XPropertyManager;
+
+public class ShootFuelCommandGroup extends CommandGroup{
```
Comment (aschokking) [inline Competition2017/src/competition/subsystems/shoot_fuel/ShootFuelCommandGroup.java]: nit, add space before ` {`

**Comment by WasabiFan** [comment]
This was merged without regard for the question of "why"... What's the use of these commands?

**Pull request merged**